### DISCUSSION TITLE
Has got "HPE_INVALID_HEADER_TOKEN" issue when trigger load testing with artillery

### DISCUSSION THREAD

**Discussion by ld0906** [artilleryio/artillery#1176] [Artillery]
this issue can be reproduced by about 95%. And when this issue happens, seem all HTTP request not triggered when add "DEBUG=http*".

Do you have met this issue before?

![image](https://user-images.githubusercontent.com/17491524/135218636-03bde782-2947-4c29-a40a-633897c2e509.png)

**Comment by hassy** [comment]
this looks like an issue with the server under test (or a proxy in between) returning an HTTP response that does not fully conform to the HTTP spec, see:

https://github.com/nodejs/node/issues/27711
